[2025-04-10T21:18:32.410+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: generate_data.run_generate_script scheduled__2023-01-24T10:00:00+00:00 [queued]>
[2025-04-10T21:18:32.448+0000] {taskinstance.py:1157} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: generate_data.run_generate_script scheduled__2023-01-24T10:00:00+00:00 [queued]>
[2025-04-10T21:18:32.453+0000] {taskinstance.py:1359} INFO - Starting attempt 1 of 2
[2025-04-10T21:18:32.549+0000] {taskinstance.py:1380} INFO - Executing <Task(PythonOperator): run_generate_script> on 2023-01-24 10:00:00+00:00
[2025-04-10T21:18:32.558+0000] {standard_task_runner.py:57} INFO - Started process 7289 to run task
[2025-04-10T21:18:32.610+0000] {standard_task_runner.py:84} INFO - Running: ['***', 'tasks', 'run', 'generate_data', 'run_generate_script', 'scheduled__2023-01-24T10:00:00+00:00', '--job-id', '568', '--raw', '--subdir', 'DAGS_FOLDER/dags_1.py', '--cfg-path', '/tmp/tmps7hf24eo']
[2025-04-10T21:18:32.648+0000] {standard_task_runner.py:85} INFO - Job 568: Subtask run_generate_script
[2025-04-10T21:18:32.941+0000] {task_command.py:415} INFO - Running <TaskInstance: generate_data.run_generate_script scheduled__2023-01-24T10:00:00+00:00 [running]> on host 7889a671eaba
[2025-04-10T21:18:33.263+0000] {taskinstance.py:1660} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='generate_data' AIRFLOW_CTX_TASK_ID='run_generate_script' AIRFLOW_CTX_EXECUTION_DATE='2023-01-24T10:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2023-01-24T10:00:00+00:00'
[2025-04-10T21:18:36.678+0000] {logging_mixin.py:151} INFO - Script Output: 
[2025-04-10T21:18:36.684+0000] {python.py:194} INFO - Done. Returned value was: None
[2025-04-10T21:18:36.731+0000] {taskinstance.py:1398} INFO - Marking task as SUCCESS. dag_id=generate_data, task_id=run_generate_script, execution_date=20230124T100000, start_date=20250410T211832, end_date=20250410T211836
[2025-04-10T21:18:36.813+0000] {local_task_job_runner.py:228} INFO - Task exited with return code 0
[2025-04-10T21:18:36.847+0000] {taskinstance.py:2776} INFO - 0 downstream tasks scheduled from follow-on schedule check
